docs/How/release_process.md

```diff
@@ -7,50 +7,73 @@
-the most recent commits and encouraging the reverse integration of all changes/fixes back into TianoCore.
+Project Mu carries a set of changes that are made on top of a given release of its upstream project,
+[TianoCore](https://www.tianocore.org/). Project Mu follows a release-based branching model with the Mu release
+corresponding to a release from the [edk2 repository](https://github.com/tianocore/edk2) in the TianoCore project.
+
+edk2 follows a [quarterly release cadence](https://github.com/tianocore/tianocore.github.io/wiki/EDK-II#stable-tags)
+with releases referred to as ["stable tags"](https://github.com/tianocore/edk2/tags). Therefore, a given Project Mu
+release branch will be named according to the stable tag it is based upon - for example, `"release/202405"` means that
+the Project Mu branch is based on the edk2 stable tag [`edk2-stable202405`](https://github.com/tianocore/edk2/releases/tag/edk2-stable202405).
+
+The release branch process keeps Project Mu up to date with TianoCore while highlighting all Project Mu differences in
+the most recent commits and encouraging the reverse integration of all changes/fixes back into TianoCore. It is worth
+noting that an effort is made to contribute relevant changes to edk2 when they are created, however, a small number of
+Project Mu changes are not intended to go to edk2.
 
 In general, the life-cycle of active code follows the following path:
 
-<center>![Release Cycle](../img/mu_release_cycle.png)</center>
-
-The Project Mu release cycle has changed with the dev/202405 and release/202405 branches to move to a more stable
-consumption pattern for downstream users.
-
-All active work in Project Mu is performed on a `dev/*` branch, named sequentially according to the date of
-TianoCore commit that it's based on (e.g. `dev/202405` is based on the `edk2-stable202405` branch in TianoCore).
-Work proceeds on that branch until a new TianoCore integration is targeted, at which point a new branch is created and
-all existing changes are rebased onto the new branch and the new branch is used for all active development going
-forward. A separate `release/*` branch is maintained, following the same naming convention (e.g. `release/202405` is the
-sibling branch to `dev/202405`). This release branch is forked off from the dev branch when stability has been reached
-and at that point only bug and security fixes will be backported to the release branch in order to maintain its
-stability. All downstream consumers should always consume from the release branch, unless they are testing out new
-functionality present in the dev branch. Project Mu will keep backporting bug and security fixes to release branches
-as part of LTS (Long-Term Support) for at least the next two releases (though it may be longer based on community
-needs). Once LTS has ended, the branch will be archived following the process described
+<center>
+

```

Comment:
Note: The empty line between the HTML tag and markdown is intentional and necessary to allow the markdown to be interpreted correctly by many markdown renderers.